x-pack/plugins/lens/public/visualizations/metric/visualization.tsx

```diff
@@ -0,0 +1,415 @@
+        arguments: {
+          metric: state.metricAccessor ? [state.metricAccessor] : [],
+          secondaryMetric: state.secondaryMetricAccessor ? [state.secondaryMetricAccessor] : [],
+          secondaryPrefix: state.secondaryPrefix ? [state.secondaryPrefix] : [],
+          max: state.maxAccessor ? [state.maxAccessor] : [],
+          breakdownBy:
+            state.breakdownByAccessor && !state.collapseFn ? [state.breakdownByAccessor] : [],
+          subtitle: state.subtitle ? [state.subtitle] : [],
+          progressDirection: state.progressDirection ? [state.progressDirection] : [],
+          color: [state.color ?? defaultColor],
+          palette: state.palette?.params
+            ? [
+                paletteService
+                  .get(CUSTOM_PALETTE)
+                  .toExpression(computePaletteParams(state.palette.params as CustomPaletteParams)),
+              ]
+            : [],
+          maxCols: [state.maxCols ?? DEFAULT_MAX_COLUMNS],
+          minTiles: maxPossibleTiles ? [maxPossibleTiles] : [],
+        },
+      },
+    ],
+  };
+};
+
+export const metricLabel = i18n.translate('xpack.lens.metric.label', {
+  defaultMessage: 'Metric',
+});
+const metricGroupLabel = i18n.translate('xpack.lens.metric.groupLabel', {
+  defaultMessage: 'Goal and single value',
+});
+
+export const getMetricVisualization = ({
+  paletteService,
+}: {
+  paletteService: PaletteRegistry;
+}): Visualization<MetricVisualizationState> => ({
+  id: LENS_METRIC_ID,
+
+  visualizationTypes: [
+    {
+      id: LENS_METRIC_ID,
+      icon: LensIconChartMetric,
+      label: metricLabel,
+      groupLabel: metricGroupLabel,
+      showExperimentalBadge: true,
+      sortPriority: 3,
+    },
+  ],
+
+  getVisualizationTypeId() {
+    return LENS_METRIC_ID;
+  },
+
+  clearLayer(state) {
+    const newState = { ...state };
+    delete newState.metricAccessor;
+    delete newState.secondaryMetricAccessor;
+    delete newState.secondaryPrefix;
+    delete newState.breakdownByAccessor;
+    delete newState.collapseFn;
+    delete newState.maxAccessor;
+    delete newState.palette;
+    // TODO - clear more?
+    return newState;
+  },
+
+  getLayerIds(state) {
+    return [state.layerId];
+  },
+
+  getDescription() {
+    return {
+      icon: LensIconChartMetric,
+      label: metricLabel,
+    };
+  },
+
+  getSuggestions,
+
+  initialize(addNewLayer, state, mainPalette) {
+    return (
+      state ?? {
+        layerId: addNewLayer(),
+        layerType: layerTypes.DATA,
+        palette: mainPalette,
+      }
+    );
+  },
+  triggers: [VIS_EVENT_TO_TRIGGER.filter],
+
+  getConfiguration(props) {
+    const hasColoring = props.state.palette != null;
+    const stops = props.state.palette?.params?.stops || [];
+    const isSupportedMetric = (op: OperationMetadata) =>
+      !op.isBucketed && supportedDataTypes.has(op.dataType);
+
+    const isSupportedDynamicMetric = (op: OperationMetadata) =>
```

Comment:
I commited this small change to block allowing the user to move static value from `maximum` to the group that doesn't support static value with dnd.

Reply:
Thank you!